package.json

```diff
@@ -7,6 +7,7 @@
     "@material-ui/core": "^1.2.3",
     "@material-ui/icons": "^1.1.0",
     "lodash": "^4.17.10",
+    "prop-types": "^15.6.2",
```

Comment:
You don't need `prop-types` because you have `Flow`. Pls, remove and use types instead